### PR TITLE
STONEBLD-871: Pipeline update pull-request note

### PR DESCRIPTION
**Pull request by Michkov**
Build service is having controller which sends Pull-requests to custom components to update tasks in pipelinerun definition. Creating note with this information.

### CODE REVIEW

File: docs/modules/ROOT/pages/how-to-guides/Secure-your-supply-chain/proc_upgrade_build_pipeline.adoc [hunk 1]
```diff
@@ -46,4 +46,8 @@ After upgrading the build pipeline, you can also customize it:
 . Find the new `.tekton` directory, which is the result of upgrading the build pipeline. 
 . Modify the contents of this directory to change how {ProductName} builds the component. 
 
-
+[NOTE]
+====
+{ProductName} creates pull requests with updates to pipelines definitions under the `.tekton` directory.
```
Comment (gtrivedi88) [inline docs/modules/ROOT/pages/how-to-guides/Secure-your-supply-chain/proc_upgrade_build_pipeline.adoc:51]: {ProductName} creates pull requests that contain updates to the pipeline definitions under the `.tekton` directory. {ProductName} recommends you allow the execution of tests on these pull requests by creating a comment containing `/ok-to-test` and then merging the changes when the tests pass.

Comment (Michkov) [reply docs/modules/ROOT/pages/how-to-guides/Secure-your-supply-chain/proc_upgrade_build_pipeline.adoc]: Updated, thanks

Comment (gtrivedi88) [reply docs/modules/ROOT/pages/how-to-guides/Secure-your-supply-chain/proc_upgrade_build_pipeline.adoc]: As per our style guide:
image

Comment (Michkov) [reply docs/modules/ROOT/pages/how-to-guides/Secure-your-supply-chain/proc_upgrade_build_pipeline.adoc]: Updated to the updated comment ;-)

**Review by gtrivedi88 — APPROVED** [review]

**Pull request merged**